sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -91,6 +91,13 @@ private void canvasRestApiCall(HttpServletRequest request,
 		 }else {
 			url=canvasURL+pathInfo;
 		}
+		String sessionId = request.getSession().getId();
+		StringBuilder strgBlder =new StringBuilder();
+		strgBlder.append("Canvas API request URL  \"");
+		strgBlder.append(url);
+		strgBlder.append("\" Session Id: ");
```

Comment:
Suggestion: I'd put the session ID before the URL since it would be easier to parse out of the log that way.